test/config_test.js

```diff
@@ -0,0 +1,73 @@
+goog.module('measurementlibrary.measure.testing.setup');
```

Comment:
I'm not sure I like 'config' for the name of the file. Why not 'setup'?

Reply:
Fixed, I had renamed the function before